src/DefinitionHelper/CreateDefinitionHelper.php

```diff
@@ -96,4 +96,16 @@ public function method(string $method, ...$arguments) : self
 
         return $this;
     }
+
+    /**
+     * Define the entry as autowired.
+     *
+     * @return self
+     */
+    public function autowire() : self
+    {
+        $this->definition->setAutowired(true);
+
+        return $this;
+    }
```

Comment:
It ist some kind of duplication, yes. Instead of this method, i could add a parameter to the constructor.